scheduler.go

```diff
@@ -310,25 +303,12 @@ func (s *Scheduler) Every(interval uint64) *Scheduler {
 	return s
 }
 
-// RunPending runs all the Jobs that are scheduled to run.
-func (s *Scheduler) RunPending() {
-	for _, job := range s.runnableJobs() {
-		s.runAndReschedule(job) // we should handle this error somehow
+func (s *Scheduler) run(job *Job) {
+	if !s.running {
```

Comment:
@JohnRoesler also, do you think this might help with that issue that we talked earlier on slack?